repositories/task_repository.py

```diff
@@ -19,3 +19,25 @@ def create(self, task: Task) -> Task:
         self.__db_context.commit()
         self.__db_context.refresh(task)
         return task
+
+    def update(self, task: Task) -> ServiceResponse:
+        content = {"msg": f"Задача по идентификатору '{task.id}' не найдена"}
+
+        db_tasks = self.__db_context.query(Task).all()
```

Comment:
А почему такой запрос? и потом такая муторная обработка ответа?


Мне кажется, так было бы и проще и лучше
```suggestion
        db_tasks = self.__db_context.query(Task).filter(Task.id == task.id)
```